### ISSUE TITLE
#define POWER_SUPPLY 0

### ISSUE THREAD

**Issue by guntherv73** [bionicbone/Marlin#8]
should use #define POWER_SUPPLY 0 in Configuration.h unless there is a hardware mod to enable the power supply
This is not there by default

**Comment by bionicbone** [comment]
Honestly I'm not sure, some say 0 some say 1. One seems more flexible for end users since if the mod is there it would work and if not, we'll then nothing will ever switch it off.

With our hardware I'm not sure it makes much difference, I certainly could not detect anything as an end user.

Do you have any detailed information?

**Comment by bionicbone** [comment]
I'll also add that the original marlin firmware (although far from a perfect example) had the setting of 1

**Comment by guntherv73** [comment]
yeah, seams to be a ramps thing mainly, found some info http://forums.reprap.org/read.php?219,132664 
https://github.com/foosel/OctoPrint/wiki/Control-your-printer's-ATX-PSU-through-a-RAMPS-board-using-OctoPrint  just mentioning seen i noticed, did not looked deeper if any actual 1284P  pin is defined when choosing melzi seen the melzi info i found never mentions any PS_ON pin or pad
